wormhole-connect/src/views/v2/Bridge/index.tsx

```diff
@@ -165,9 +165,14 @@ const Bridge = () => {
       setSelectedRoute('');
     } else {
       const autoselectedRoute = route || validRoutes[0].route.name;
+      const isSelectedRouteValid =
+        validRoutes.findIndex((r) => r.route.name === selectedRoute) > -1;
```

Comment:
I think we need to include `selectedRoute` as a dependency

Reply:
Actually, we don't want to run this side-effect whan selected route changes. Only when the `route` has an its initial value or when we have the list of supported routes, so we can initially select a route among them. I tried to explain that in the comment above the side-effect, but please let me know if it's not clear 🙏 
```
// Set selectedRoute if the route is auto-selected
// After the auto-selection, we set selectedRoute when user clicks on a route in the list
```

Reply:
Makes sense to me. We should really enable the linter check for hook dependencies and add an exception for this dependency (+ document why), but that's out of scope :)